update.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# the below will work assuming no other custom modifications have been made.
+rm -rf node_modules/
+git checkout package.json package-lock.json
```

Comment:
can you please explain, why to checkout this files first, and then pull the changes afterwards? Shouldnt both files be updated on a pull as well?

Reply:
I think this is mainly to make sure there are no local changes to these files that would mess up a `git pull`

Reply:
@Shawn8901 you can't pull if modifications have been made. npm update & dedupe will modify both.

<3